roles-worker.tf

```diff
@@ -84,3 +86,33 @@ resource "aws_iam_role_policy_attachment" "quortex-autoscaler-policy-attach" {
   role       = aws_iam_role.quortex_role_worker.name
   policy_arn = aws_iam_policy.quortex-autoscaler-policy.arn
 }
+
+### Attach a new policy for the cloudwatch-exporter to the worker role
+
+resource "aws_iam_policy" "quortex-cloudwatch-policy" {
+  description = "Allow the cloudwatch-exporter to make calls to the AWS CloudWatch APIs."
+
+  policy = <<POLICY
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Action": [
+                "cloudwatch:ListMetrics",
+                "cloudwatch:GetMetricStatistics",
+                "cloudwatch:GetMetricData",
+                "tag:GetResources"
+            ],
+            "Resource": "*", 
+            "Effect": "Allow"
+        }
+    ]
+}
+POLICY
+# TODO: can we limit to the Resources created by this deployment ?
```

Comment:
Can we limit to the Resources created by this deployment ?

Reply:
No, we cannot.